resources/assets/less/variables.less

```diff
@@ -91,6 +91,7 @@
 @font-size--title-small: 14px;
 @font-size--title-small-2: 15px;
 @font-size--title-small-3: 16px;
+@font-size--title-small-4: 17px;
```

Comment:
see if smaller font size works as well because we already have too many sizes here.